### ISSUE TITLE
how to install this software in fedora?

### ISSUE THREAD

**Issue by zjhxmjl** [getlantern/lantern#2180]
Hi,@atavism is there any guides for how to install lantern in fedora 20?thanks a million!

**Comment by myleshorton** [comment]
Thanks so much for your interest @zjhxmjl! At this point we only Ubuntu and have spare resources to support Fedora in the near term. We are definitely open to patches, however, if you see an easy way to add fedora support. Thanks again.

**Comment by aranhoide** [comment]
However, users of other GNU/Linux distributions should be able to build from source (see the README in this repo).

**Comment by yejingchen** [comment]
Not really. I use Fedora 23 and when running `make docker` (which is the first process described in README) it tried to run yum and install fedora 21 packages, which is certainly will break my system.

**Comment by xiam** [comment]
Oh yeah, it does run `yum` within the docker container. Your OS is not affected at all.

**Comment by yejingchen** [comment]
Alright, I apologize for my ignorance... I'll give it a try.

**Comment by Nearsc** [comment]
I have also tried to install lantern on Fedora 20. I just follow the steps in *`README.md`*, and type **`make lantern`**. Then I got the executable file *`lantern`*. I type **`./lantern'**, and a web appear showing the lantern version is 2.1.1. But I still can't get access to some websites such as twitter.
